compat/win32/fscache.c

```diff
@@ -257,7 +252,7 @@ static void fscache_clear(void)
 /*
  * Checks if the cache is enabled for the given path.
  */
-static inline int fscache_enabled(const char *path)
+int fscache_enabled(const char *path)
```

Comment:
This hunk and the one above look like a good change in its own right. Maybe that merits a separate commit, too?